launch files refer to older vlam node names

I have been following the all your provided steps to install flock2. 

> mkdir -p ~/flock2_ws/src
cd ~/flock2_ws/src
git clone https://github.com/clydemcqueen/flock2.git
git clone https://github.com/ptrmu/fiducial_vlam.git
git clone https://github.com/clydemcqueen/tello_ros.git
cd ..
source /opt/ros/eloquent/setup.bash
If you didn't install Gazebo, avoid building tello_gazebo:
colcon build --event-handlers console_direct+ --packages-skip tello_gazebo
> 

Upon the last instruction of the Step 4. I faced an error shown below. 

~/flock2_ws$ colcon build --event-handlers console_direct+
[0.542s] ERROR:colcon:colcon build: Duplicate package names not supported:
- tello_description:
  - src/tello_ros/tello_description
  - tello_ros/tello_description
- tello_driver:
  - src/tello_ros/tello_driver
  - tello_ros/tello_driver
- tello_gazebo:
  - src/tello_ros/tello_gazebo
  - tello_ros/tello_gazebo
- tello_msgs:
  - src/tello_ros/tello_msgs
  - tello_ros/tello_msgs

Thanks for the swift answer. I have managed to fix that like ur solution stated above. However, it get another error as shown below. 

> flock2
CMake Error at CMakeLists.txt:49 (find_package):
  By not providing "Findtello_msgs.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "tello_msgs", but CMake did not find one.
  Could not find a package configuration file provided by "tello_msgs" with
  any of the following names:
    tello_msgsConfig.cmake
    tello_msgs-config.cmake
  Add the installation prefix of "tello_msgs" to CMAKE_PREFIX_PATH or set
  "tello_msgs_DIR" to a directory containing one of the above files.  If
  "tello_msgs" provides a separate development package or SDK, be sure it has
  been installed.>

Please advice and thank you for the help.

Thank you. I have managed to delete and recreating the workspace building was fine as well.. However, when I am trying to run launch_one.py, I would face some error message. `[ERROR] [launch]: Caught exception in launch (see debug for traceback): executable 'vmap_node' not found on the libexec directory '/home/user/flock2_ws/install/fiducial_vlam/lib/fiducial_vlam' ` Pondering into the file system, I couldnt find any references of vmap_node.

--- a/launch/gazebo_launch.py
+++ b/launch/gazebo_launch.py
@@ -45,8 +45,8 @@ def generate_launch_description():
         ], output='screen'),
 
         # Load and publish a known map
-        Node(package='fiducial_vlam', node_executable='vmap_node', output='screen',
-             node_name='vmap_node', parameters=[{
+        Node(package='fiducial_vlam', node_executable='vmap_main', output='screen',
+             node_name='vmap_main', parameters=[{
                 'use_sim_time': True,                           # Use /clock if available
                 'publish_tfs': 1,                               # Publish marker /tf
                 'marker_length': 0.1778,                        # Marker length
@@ -96,8 +96,8 @@ def generate_launch_description():
 
             # Localize this drone against the map
             # Future: need odometry for base_link, not camera_link
-            Node(package='fiducial_vlam', node_executable='vloc_node', output='screen',
-                 node_name='vloc_node', node_namespace=namespace, parameters=[{
+            Node(package='fiducial_vlam', node_executable='vloc_main', output='screen',
+                 node_name='vloc_main', node_namespace=namespace, parameters=[{
                     'use_sim_time': True,                           # Use /clock if available
                     'publish_tfs': 1,                               # Publish drone and camera /tf
                     'stamp_msgs_with_current_time': 0,              # Use incoming message time, not now()

--- a/launch/launch_one.py
+++ b/launch/launch_one.py
@@ -33,11 +33,11 @@ def generate_launch_description():
              node_name='drone_base', node_namespace='solo'),
 
         # Mapper
-        Node(package='fiducial_vlam', node_executable='vmap_node', output='screen'),
+        Node(package='fiducial_vlam', node_executable='vmap_main', output='screen'),
 
         # Visual localizer
-        Node(package='fiducial_vlam', node_executable='vloc_node', output='screen',
-             node_name='vloc_node', node_namespace='solo'),
+        Node(package='fiducial_vlam', node_executable='vloc_main', output='screen',
+             node_name='vloc_main', node_namespace='solo'),
 
         # WIP: planner
         Node(package='flock2', node_executable='planner_node', output='screen'),

--- a/launch/launch_two.py
+++ b/launch/launch_two.py
@@ -9,26 +9,26 @@ from launch.actions import ExecuteProcess
 
 
 def generate_launch_description():
-    urdf1 = os.path.join(get_package_share_directory('tello_description'), 'urdf', 'drone_1.urdf')
-    urdf2 = os.path.join(get_package_share_directory('tello_description'), 'urdf', 'drone_2.urdf')
+    urdf1 = os.path.join(get_package_share_directory('tello_description'), 'urdf', 'tello_1.urdf')
+    urdf2 = os.path.join(get_package_share_directory('tello_description'), 'urdf', 'tello_2.urdf')
 
     dr1_ns = 'drone1'
     dr2_ns = 'drone2'
 
     dr1_params = [{
         'drone_ip': '192.168.86.206',
-        'command_port': '11001',
-        'drone_port': '12001',
-        'data_port': '13001',
-        'video_port': '14001'
+        'command_port': 11001,
+        'drone_port': 12001,
+        'data_port': 13001,
+        'video_port': 14001
     }]
 
     dr2_params = [{
         'drone_ip': '192.168.86.212',
-        'command_port': '11002',
-        'drone_port': '12002',
-        'data_port': '13002',
-        'video_port': '14002'
+        'command_port': 11002,
+        'drone_port': 12002,
+        'data_port': 13002,
+        'video_port': 14002
     }]
 
     base_params = [{
@@ -75,11 +75,11 @@ def generate_launch_description():
              node_name='base2', node_namespace=dr2_ns),
 
         # Mapper
-        Node(package='fiducial_vlam', node_executable='vmap_node', output='screen'),
+        Node(package='fiducial_vlam', node_executable='vmap_main', output='screen'),
 
         # N visual localizers
-        Node(package='fiducial_vlam', node_executable='vloc_node', output='screen',
+        Node(package='fiducial_vlam', node_executable='vloc_main', output='screen',
              node_name='vloc1', node_namespace=dr1_ns),
-        Node(package='fiducial_vlam', node_executable='vloc_node', output='screen',
+        Node(package='fiducial_vlam', node_executable='vloc_main', output='screen',
              node_name='vloc2', node_namespace=dr2_ns),
     ])

